cmd/create.go

```diff
@@ -425,8 +425,8 @@ func createHost(provider, name, region, zone, projectID, userData, inletsPort st
 	} else if provider == EquinixMetalProvider {
 		return &provision.BasicHost{
 			Name:     name,
-			OS:       "ubuntu_16_04",
-			Plan:     "t1.small.x86",
+			OS:       "ubuntu_20_04",
+			Plan:     "c3.small.x86",
```

Comment:
Can you take this as a flag? `--plan`

Use the default, unless someone has given an override. You'll see how I did it for the operator.

Reply:
Is this not already in place with -> https://github.com/inlets/inletsctl/blob/ed0bcc767eb9357bef75062dee746684b97eb0bf/cmd/create.go#L277

The default is the on writen, and after the hostReq get checked for planoverwrite?

Reply:
good call the t1 instances will be deprecated

Reply:
OK we are doing that in a weird non-obvious way, I'd much prefer to determine the plan in this function than to mutate the result of the function from a command line flag after the fact.

That's why I missed this. I'll merge this, but would you like to work on the proposed behaviour change? That's how the operator works ATM:

https://github.com/inlets/inlets-operator/blob/master/controller.go#L553-L569

Reply:
@dirien could you refactor how we handle the plan as per the operator?

Reply:
Created a issue for it #111